tests/vmi_networking_test.go

```diff
@@ -722,42 +722,46 @@ var _ = Describe("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 			Expect(err).ToNot(HaveOccurred())
 			Expect(len(serverVMI.Status.Interfaces)).To(Equal(1))
 
-			serverIP := serverVMI.Status.Interfaces[0].IP
-			if netutils.IsIPv6String(serverIP) {
-				By("Checking traceroute from vmi to cluster nodes gateway")
-				// Cluster nodes subnet (docker network gateway)
-				// Docker network subnet cidr definition https://bit.ly/2wZTgMK
-				err := tests.CheckForTextExpecter(serverVMI, createExpectTraceroute6("2001:db8:1::1"), 30)
-				Expect(err).ToNot(HaveOccurred(), "Failed to traceroute to VMI %s within the given timeout", serverVMI.Name)
-			} else {
-				if ipv4NetworkCIDR == "" {
-					ipv4NetworkCIDR = api.DefaultVMCIDR
+			for _, serverIP := range serverVMI.Status.Interfaces[0].IPs {
+				if netutils.IsIPv6String(serverIP) {
+					Skip("Skip till https://github.com/kubevirt/kubevirtci/pull/432 is merged")
```

Comment:
https://github.com/kubevirt/kubevirtci/pull/432 was merged.

Reply:
ye but there was a kubevirt bump ?
and its the only one we need ? i wanted to make sure manually that it works before, and see what PRs need to be merged (for example vm scheme), before i remove the hold

Reply:
it doesnt work with the following
VM scheme PR
bump to have 432 in kubevirt itself
remove the skip of 432

we need https://github.com/kubevirt/kubevirt/pull/3957 at least
so we either wait for it, or add the skip according that one.
i vote for the 2nd, i dont see a reason to wait if its ok please.

Reply:
We should wait until #3957, #4000 and #3958 are merged. Meanwhile, this PR can depend on those PRs.

Reply:
why should we wait ? usually we just add a skip until and do move forward.

Reply:
Because we do not know if it works or not, or better phrase it as: "we do not **add** dead code".

Reply:
I meant that i will check locally, be sure that it works, as i did before and stopped when i found something that changed the story. 
but nvm